### DISCUSSION TITLE
visualization of particles

### DISCUSSION THREAD

**Discussion by govindiitd** [techMathGroup/openHFDIB-DEM#12] [General]
I read the to visualize the particle, two python files are required.  But I don't find them in case folders. 

Can anyone please guide me through this?

Regards,
Govind

**Comment by MartinIsoz** [comment]
Dear Govind,

the python scripts used in visualization of wiki cases, e.g.

https://github.com/techMathGroup/openHFDIB-DEM/wiki/Tutorial:-incompressible%E2%80%90%E2%80%90pimpleHFDIBFoam%E2%80%90%E2%80%90fallingParticleDistribution#results-visualization

are within the case folders, that is

https://github.com/techMathGroup/openHFDIB-DEM/tree/main/tutorials/incompressible/pimpleHFDIBFoam/fallingParticleDistribution

for this specific tutorial.

The python scripts are present only in the two folders linked to the
cases from wiki. The wiki itself is a quite recent addition and as a
small research team, we were not yet able to extend it to all the
examples and tutorials on github. But it should be possible to copy the
python scripts from the wiki-cases to any other case and prepare the
postprocessing using the wiki description.

I hope that this will help.
